fastseq/optimizer/fairseq/generate.py

```diff
@@ -60,7 +63,7 @@ def __init__(self, args, task, message_queue):
 
         self.args = args
         self.message_queue = message_queue
-        self.has_target = False
+        self.has_target = True
```

Comment:
Why we need change this?

Reply:
I changed for experimentation, reverted  back